netlify.toml

```diff
@@ -1,13 +1,23 @@
 [build.environment]
-  NODE_ENV = "production"
+  NODE_ENV     = "production"
   NODE_VERSION = "14"
-  NPM_VERSION = "7"
+  NPM_VERSION  = "7"
 
+# Deploy previews are generated from pull requests
+[context.deploy-preview]
+  command = "npm run prettier && npm run eslint && npm run build"
+
+# Running Prettier and ESLint for production deploys would be redundant
+# because we have already run them for deploy previews.
+# Plus by not running them we might save a few seconds per deploy =D
```

Comment:
Can confirm with a sample size of one (the quotes are from deploy logs):

- Deploy preview: "build.command completed in 6.4s"
- Production deploy: "build.command completed in 2.2s"

😎